app.js

```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const hbs = require('hbs');
 const path = require('path');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
+const { response } = require('express');
```

Comment:
this const is used to load the express library, and doesn't need to be deconstructed. Is there a reason why you used it like this? was it required by vscode?